### ISSUE TITLE
How to write in xml format the hierarchies like subtype, supertype??

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ontopia/ontopia#226]
```
I am in the middle of creating some examples of reification in Topic Map by
using xtm file format. One major problem with using xtm format for already
created topic map through omnigator editor is it looses the clean and neat
understandable format of topic map which we can see before uploading it for
editor. Instead of topics it shows numbers and ids. So I started to create
another small topic map directly in xtm format in the note pad. I could
create reification as well. Now, I want to create supertype, subtype
hierarchies so that I can use them. I tried to create association of the
type supertype but how to get the role types like Parent Role and Child
Role which are not considered in normal role types?

In short, I want to write in xml format the hierarchies like subtype,
supertype in my small topic map in xtm format without using an editor. How
to do it?

```

Original issue reported on code.google.com by `aparnala...@gmail.com` on 31 Mar 2010 at 7:16
